### DISCUSSION TITLE
how do you change x-frame-options?

### DISCUSSION THREAD

**Discussion by bogganj94** [TriliumNext/Trilium#1953] [Q&A]
OS:Windows 10

Method i used to install:
pulled from master, npm install

i have a website i am working on in xampp and i am trying to figure out how to embed trillium on my home page. i have a iframe switcher used for the main body content. when i add localhost:8080 to an iframe i get an error saying the x-frame-options is set to SAMEORIGIN. is there a way to change this so i can view trillium in a iframe outside of the nodejs server?

if you need more info let me know. and im sorry if im asking this in the wrong section i figured since its a really minor issue id just ask here.

**Comment by meichthys** [comment]
This would also be useful for embedding Trilium into Nextcloud using Nextclou's External-Sites app.

**Comment by zadam** [comment]
Trilium doesn't have support for this, on the code level this could be done by changing helmet configuration.

Otherwise a reverse proxy could be configured to change the header.

**Reply by meichthys** [reply]
Works great. Thanks!
![image](https://user-images.githubusercontent.com/10717998/176578851-fe84ef9d-0ce5-4429-b334-f40f782ccb56.jpeg)
